Game2048Base/src/main/java/models/grid2048/Grid2048.java

```diff
@@ -0,0 +1,118 @@
+package models.grid2048;
+
+import com.google.common.annotations.VisibleForTesting;
+import javalib.worldimages.OutlineMode;
+import javalib.worldimages.OverlayOffsetImage;
+import javalib.worldimages.Posn;
+import javalib.worldimages.RectangleImage;
+import javalib.worldimages.WorldImage;
+import models.square.EmptySquare;
+import models.square.Square;
+import models.square.Tile;
+import utility.PosnUtility;
+import java.awt.Color;
+import java.util.ArrayList;
+import java.util.HashSet;
+
+public class Grid2048 {
+
+    public Square[][] grid;
+    static WorldImage IMAGE = new RectangleImage(
+            Square.SIDE_LENGTH * 4,
```

Comment:
There are a lot of "magic" `4`s in this code that all refer to how many squares fit in a row. Can we pull that out into `SQUARES_PER_AXIS` or something like that?